CONTRIBUTING.MD

```diff
@@ -64,8 +64,13 @@ The first line is the subject and should be no longer than 70 characters, the se
 
 Most contributions involve some sort of documentation. See our [doc contribution guidelines][doc-contrib] for details.
 
+## Changelog
+
+The operator-sdk maintains a changelog and most contributions should involve a changelog fragment following the following [template][changelog-template]
+
 [operator_framework]: https://groups.google.com/forum/#!forum/operator-framework
 [developer-guide]: https://sdk.operatorframework.io/docs/contribution-guidelines/developer-guide/
 [reporting-issues]: https://sdk.operatorframework.io/docs/contribution-guidelines/reporting-issues/
 [golang-style-doc]: https://github.com/golang/go/wiki/CodeReviewComments
-[doc-contrib]:https://sdk.operatorframework.io/docs/contribution-guidelines/documentation
+[doc-contrib]: https://sdk.operatorframework.io/docs/contribution-guidelines/documentation
+[changelog-template]: https://github.com/operator-framework/operator-sdk/blob/master/changelog/fragments/00-template.yaml
```

Comment:
This will be broken in this PR but pass once it's merged.
```suggestion
[changelog-contrib]:https://master.sdk.operatorframework.io/docs/contribution-guidelines/changelog
```